GitUI/UserControls/FileStatusList.cs

```diff
@@ -726,12 +726,16 @@ ObjectId GetRevisionOrHead(ObjectId rev, Lazy<ObjectId> head)
             }
         }
 
-        public void SetDiffs(GitRevision selectedRev = null, GitRevision parentRev = null, IReadOnlyList<GitItemStatus> items = null)
+        public void SetDiffs([CanBeNull] GitRevision selectedRev, [CanBeNull] GitRevision parentRev, [NotNull] IReadOnlyList<GitItemStatus> items)
         {
             Revision = selectedRev;
-            GitItemStatusesWithDescription = items == null
-                ? Array.Empty<(GitRevision, string, IReadOnlyList<GitItemStatus>)>()
-                : new[] { (parentRev, _diffWithParent.Text + GetDescriptionForRevision(parentRev?.ObjectId), items) };
+            GitItemStatusesWithDescription = new[] { (parentRev, _diffWithParent.Text + GetDescriptionForRevision(parentRev?.ObjectId), items) };
+        }
+
+        public void SetDiffs()
```

Comment:
Should it rather be renamed to `ClearDiffs` or `ResetDiffs`?

Reply:
ClearFileStatus()
I did not want to change so much in this PR - will change this and open a new PR for Stash

Reply:
It should be easier to get smaller PRs merged, but with @RussKie you never know. The smaller the PRs are, the more changes requested per line changed. There is a review queue...
But many of my open PRs are related to each other, I am blocked and will probably spend time on other projects now.

Reply:
> But the counterpart is named SetDiffs

The method could maybe be called Clear() instead
SetDiffs() is not a good name, but I do not have any better suggestion

Reply:
As mentioned in #7938, changing to ClearDiffs()